BFF.py

```diff
@@ -9,10 +9,12 @@
 import requests
 import json
 from flask import Flask, jsonify, request
+from flask_cors import CORS
```

Comment:
I understand the addition of CORS capability here, although I'm not familiar with this library.
My opinion (and practice when I work in the service layer) is that CORS policy should be the responsibility of the nginx proxy, and that service-level http servers should be agnostic.
One reason is that service http CORS header fields can interfere with nginx CORS (although there are techniques for removing any CORS header fields in nginx). Also, as a general rule, we shouldn't rely on service CORS because each service does it differently!
Anyway, my 2c!

Reply:
That makes sense. I am not using nginx locally, so I will modify this based on an environment variable like so:
```python
if os.environ.get('FLASK_ENV'):
    from flask_cors import CORS
    CORS(app)
```

How do you feel about that?